rosa_architecture/rosa_policy_service_definition/rosa-sre-access.adoc

```diff
@@ -16,15 +16,26 @@ For a list of the available subprocessors, see the link:https://access.redhat.co
 
 include::modules/sre-cluster-access.adoc[leveloffset=+1]
 include::modules/rosa-red-hat-support-access.adoc[leveloffset=+1]
+ifdef::openshift-rosa,openshift-rosa-hcp[]
+[role="_additional-resources"]
+.Additional resources
+* For more information, see xref:../../support/approved-access.adoc#approved-access[Approved Access].
+
+* For more information, see xref:../../authentication/using-rbac.adoc#default-roles_using-rbac[Default cluster roles]
+endif::openshift-rosa,openshift-rosa-hcp[]
```

Comment:
**If** the only places this assembly is used are ROSA and ROSA HCP, I don't think you need the conditional. I am not sure how to verify that though.
```suggestion
[role="_additional-resources"]
.Additional resources
* For more information, see xref:../../support/approved-access.adoc#approved-access[Approved Access].

* For more information, see xref:../../authentication/using-rbac.adoc#default-roles_using-rbac[Default cluster roles]
```